spec/support/api/schemas/institutions.json

```diff
@@ -20,7 +20,9 @@
           }
         }
       },
-      "required": ["version"]
```

Comment:
do these changes have anything to do with archiving institutions?

Reply:
bad merge possibly, will cleanup

Reply:
:heavy_plus_sign: to cleanup

Reply:
looks like this is due to linter rules i have for json files in my IDE